app/src/main/res/layout/app_bar.xml

```diff
@@ -19,7 +19,7 @@
             android:layout_height="match_parent"
             android:layout_margin="10dp"
             android:layout_weight="1"
-            android:background="@color/white"
```

Comment:
I think you should revert this whole file.

It looks very weird.

![image](https://user-images.githubusercontent.com/23643192/113225325-5de68600-925b-11eb-995d-326b41052625.png)


I reverted this file and it looks decent enough.

![image](https://user-images.githubusercontent.com/23643192/113225374-78206400-925b-11eb-919d-a5d2f59e68ba.png)